deepspeed/module_inject/auto_tp.py

```diff
@@ -279,40 +282,27 @@ def tp_parser(model):
         module_list = AutoTP.get_module_list(model)
         assert AutoTP.supported(model), "AutoTP not supported for model. Please use kernel injection since container policy for model exists." \
         if AutoTP.kernel_supported(module_list) else "AutoTP not supported for model. Please provide policy."
-        norm_layer_name_list = ['LayerNorm', 'layer_norm', 'ln_1', 'ln_2']
-        #ln_1 , ln_2 for Qwen
+
+        allReduceLinearItems = os.environ['allReduceLinearItems']
```

Comment:
Could the environment keep DeepSpeed as `DS_ALL_REDUCE_LINEAR_ITEMS`?  This will make the naming more consistent with existing DeepSpeed knobs.